frontend/.eslintrc.js

```diff
@@ -141,6 +148,9 @@ module.exports = {
             "semi": "off",
             "no-unused-vars": "off",
             "no-extra-parens": "off",
+
+            // Change strange defaults.
+            "@typescript-eslint/space-before-function-paren": ["error", "never"],
```

Comment:
The proper idiomatic config would be

```
{"anonymous": "always", "named": "never", "asyncArrow": "always"}
```

IMO. Plus, we also need to configure the base ESLint rule, not just the one in the TS-Plugin.

Reply:
Should be fixed